tests/test_qa_v1_participant_heal_retry.py

```
@@ -0,0 +1,688 @@
+        )
+    _assert_inworld_escalate(str(ei.value), "不存在之人甲")
+    # 输入文未改（调用方仍持有原 text）
+    assert text == "着不存在之人甲核清太仓，边饷优先"
+    # 不得落库
+    session = GameSession.__new__(GameSession)
+    session.db = db
+    session.state = state
+    session.llm_config = None
+    session.content = content
+    before = len(session.db.list_directives(state) or [])
+    # 无 payload 可加；确认无因副作用落行
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)